### DISCUSSION TITLE
Overriding ASSET_URL for certain assets like Vite

### DISCUSSION THREAD

**Discussion by sts-ryan-holton** [laravel/framework#50424] [Q&A]
Hi, I'm building two Laravel applications. One is exclusively a backend containing various images in my public directory, the other (and potentially more) is the front-end blog, the front-end needs to pull assets from the other project, so setting `ASSET_URL` to the other Laravel project works, except now my front-end assets (CSS & JS from Vite)for this project are also pulling from there which I don't want.

Basically, how do I:

- Laravel project 1
  - CSS: this project
  - JS: this project
  - IMAGES: Laravel project 2
- Laravel project 2
  - CSS: this project
  - JS: this project
  - IMAGES: this project

Using `asset()` as a helper I feel is conventional, but somehow need to override it for Vite css and js etc.
